MultisigTests/Features/Security/KeychainItemStoreTests.swift

```diff
@@ -20,61 +21,61 @@ class KeychainItemStoreTests: XCTestCase {
 
     public override func tearDown() {
         super.tearDown()
-        try! kciStore.delete(.generic(id: SensitiveStore.derivedPasswordTag, service: ProtectionClass.sensitive.service()))
-        try! kciStore.delete(.generic(id: SensitiveStore.sensitiveEncryptedPrivateKeyTag, service: ProtectionClass.sensitive.service()))
-        try! kciStore.delete(.ecPubKey())
-        try! kciStore.delete(.enclaveKey())
+        try! kciStore.delete(kciFactory.generic(dataId: DataID(id: ProtectedKeyStore.derivedPasswordTag)))
+        try! kciStore.delete(kciFactory.generic(dataId: DataID(id: ProtectedKeyStore.encryptedPrivateKeyTag)))
+        try! kciStore.delete(kciFactory.ecPubKey())
+        try! kciStore.delete(kciFactory.enclaveKey())
     }
 
     func testDeleteItem() throws {
         // Given
-        let randomKey = try kciStore.create(KeychainItem.ecKeyPair) as! SecKey
+        let randomKey = try kciStore.create(kciFactory.ecKeyPair()) as! SecKey
         let randomPublicKey = SecKeyCopyPublicKey(randomKey)!
-        XCTAssertTrue(try kciStore.find(KeychainItem.ecPubKey()) == nil, "Precondition failed: Keychain not empty!!")
-        try kciStore.create(.ecPubKey(publicKey: randomPublicKey))
+        XCTAssertTrue(try kciStore.find(kciFactory.ecPubKey()) == nil, "Precondition failed: Keychain not empty!!")
+        try kciStore.create(kciFactory.ecPubKey(publicKey: randomPublicKey))
 
         // When
-        try kciStore.delete(.ecPubKey())
+        try kciStore.delete(kciFactory.ecPubKey())
 
         //Then
-        XCTAssertTrue(try kciStore.find(KeychainItem.ecPubKey()) == nil, "Delete item failed")
+        XCTAssertTrue(try kciStore.find(kciFactory.ecPubKey()) == nil, "Delete item failed")
     }
 
     func testDeleteItemIgnoreNotFound() throws {
         // Given
-        XCTAssertTrue(try kciStore.find(KeychainItem.ecPubKey()) == nil, "Precondition failed: Keychain not empty!!")
+        XCTAssertTrue(try kciStore.find(kciFactory.ecPubKey()) == nil, "Precondition failed: Keychain not empty!!")
 
         // When
-        try kciStore.delete(.ecPubKey())
+        try kciStore.delete(kciFactory.ecPubKey())
 
         //Then
-        XCTAssertTrue(try kciStore.find(KeychainItem.ecPubKey()) == nil, "Delete item failed")
+        XCTAssertTrue(try kciStore.find(kciFactory.ecPubKey()) == nil, "Delete item failed")
     }
 
     func testStoreAndRetrievePasscode() throws {
         // Given
         let randomString = UUID().uuidString
-        let passCodeData = try kciStore.find(KeychainItem.generic(id: SensitiveStore.derivedPasswordTag, service: ProtectionClass.sensitive.service())) as! Data?
+        let passCodeData = try kciStore.find(KeychainItem.generic(id: ProtectedKeyStore.derivedPasswordTag, service: ProtectionClass.sensitive.service())) as! Data?
```

Comment:
it is confusing that ProtectedKeyStore items are used in this test case and not in ProtectedKeyStoreTests.